loopd/swapclient_debug_server.go

```diff
@@ -0,0 +1,25 @@
+// +build dev
```

Comment:
I think the "idiomatic" naming of the file would be `swapclient_server_debug.go`, the build tag suffix usually comes last. Alternatively we could just add this method to `swapclient_server.go`. It wouldn't be used if not registered. But I guess that comes down to personal preference.

Reply:
Renamed and moved function + macaroons into `swapclient_server_debug.go`.